### PR TITLE
Feature/account get

### PR DESCRIPTION
**Pull request by YiFeiZhang2**

### CODE REVIEW

File: controllers/account.controller.js [hunk 1]
```diff
@@ -30,6 +30,31 @@ async function getUserByEmail(req, res) {
     }
 }
 
+/**
+ * @async
+ * @function getUserById
+ * @param req
```
Comment (pierreTklein) [inline controllers/account.controller.js]: What are the types for these objects?

File: controllers/account.controller.js [hunk 2]
```diff
@@ -30,6 +30,31 @@ async function getUserByEmail(req, res) {
...
+    const acc = await Services.Account.findById(req.params.id);
+    
+    if (acc) {
+        return res.status(200).json({
+            message: "Account found by user id",
+            data: acc.toStrippedJSON()
+        });
+    } else {
+        // tentative error code
+        return res.status(400).json({
```
Comment (pierreTklein) [inline controllers/account.controller.js]: 404

**Comment by pierreTklein** [comment]
I'm confused, because travis-ci isn't triggering for this PR

**Comment by pierreTklein** [comment]
Or maybe it is?

**Comment by YiFeiZhang2** [comment]
I see an 'All checks have passed' message. What does it look like on your end?

File: controllers/account.controller.js (hunk 1 above)
Comment (YiFeiZhang2) [reply controllers/account.controller.js]: Done

Comment (YiFeiZhang2) [reply controllers/account.controller.js]: I'll also fix this issue on the other PR

Comment (YiFeiZhang2) [reply controllers/account.controller.js]: I'll update req and res for all the controllers.

File: controllers/account.controller.js (hunk 2 above)
Comment (YiFeiZhang2) [reply controllers/account.controller.js]: Done

**Pull request merged**